python-package/README.rst

```diff
@@ -195,27 +191,37 @@ Use this option to make LightGBM output time costs for different internal routin
 Install from `conda-forge channel <https://anaconda.org/conda-forge/lightgbm>`_
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-If you use ``conda`` to manage Python dependencies, you can install LightGBM using ``conda install``.
+``lightgbm`` conda packages are available from the ``conda-forge`` channel.
+
+.. code:: sh
+
+    conda install -c conda-forge lightgbm
+
+The are precompiled packages that are fast to install.
```

Comment:
Completely not sure, just want to bring your attention to a possible typo.
```suggestion
These are precompiled packages that are fast to install.
```

Reply:
You are right, thanks for catching it.